### PR TITLE
[stable8] Redirect page if not shown in an iframe

### PR DESCRIPTION
**Pull request by LukasReschke**

### CODE REVIEW

**Comment by PVince81** [comment]
:+1:

File: appinfo/version [hunk 1]
```diff
@@ -1 +1 @@
-0.7
+0.7.1
```
Comment (nickvergessen) [inline appinfo/version]: same version as in 8.1 :speak_no_evil:

**Comment by nickvergessen** [comment]
:+1:

**Comment by davitol** [comment]
:+1:

**Pull request merged**